### PR TITLE
Replace eltwise opt kernels with eltwise_blocked_opt

### PR DESCRIPTION
**Pull request by byungilm**
Signed-off-by: Min, Byungil <byungil.min@intel.com>

### Details:
- Removed deprecated kerenel
- Replaced eltwise_b_fs_yx_fsv4 with eltwise_blocked_opt

### Tickets:
 - 74370

### CODE REVIEW

**Comment by isanghao** [comment]
No regression from discrete gpu:  --this ../log/2023-02-09--04-41.report --ref ../log/2023-02-08--13-54.report

File: src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_vload8.cpp [hunk 1]
```diff
@@ -141,7 +143,7 @@ KernelsData EltwiseKernel_vload8::GetKernelsData(const Params& params, const opt
     return {kd};
 }
 
-KernelsPriority EltwiseKernel_vload8::GetKernelsPriority(const Params& /*params*/, const optional_params& /*options*/) const {
-    return FORCE_PRIORITY_8;
+KernelsPriority EltwiseKernel_vload8::GetKernelsPriority(const Params& params, const optional_params& /*options*/) const {
+    return FORCE_PRIORITY_1;
```
Comment (isanghao) [inline src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_vload8.cpp:147]: why did you change priority of this kernel?

**Review by isanghao — COMMENTED** [review]
generally looks good. minor comment left. Please check performance on gen9 and gen12lp.

File: src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_vload8.cpp (hunk 1 above)
Comment (byungilm) [reply src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_vload8.cpp:147]: if the model has no broadcasting, no multiple post-ops and divided by 8 in feature axis. I thought this kernel would have the best perf with the given lws (except b_fs_yx_fsv16 with sub_group_block). I tested this with unit-tests of fs_b_yx_fsv32. 
Is there a point to check? or Is there anything I missed about this change?

**Comment by byungilm** [comment]
deprecated PR (replaced by https://github.com/openvinotoolkit/openvino/pull/15717)
